media/transcoder/get_job.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python
+
+# Copyright 2020 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Google Cloud Transcoder sample for getting the details for a job.
+
+Example usage:
+    python get_job.py --project-id <project-id> --location <location> --job-id <job-id>
+"""
+
+import argparse
+
+from google.cloud.video.transcoder_v1beta1.services.transcoder_service import (
+    TranscoderServiceClient,
+)
+
+# [START transcoder_get_job]
+
+
+def get_job(project_id, location, job_id):
+    """Gets a job.
+
+    Args:
+        project_id: The GCP project ID.
+        location: The location this job is in.
+        job_id: The job ID."""
+
+    client = TranscoderServiceClient()
+
+    name = f"projects/{project_id}/locations/{location}/jobs/{job_id}"
+    response = client.get_job(name=name)
+    print(f"Job: {response.name}")
+    return response
+
+
+# [END transcoder_get_job]
+
+
+def get_job_state(project_id, location, job_id):
```

Comment:
This isn't called. Perhaps it should be deleted, or a call to it should be added to the main section

Reply:
I see this is for the next example, so it should be deleted from this file.

Reply:
Removed